ziti-agent/cmd/webhook/pods.go

```diff
@@ -294,20 +304,12 @@ func (zh *zitiHandler) handleTunnelCreate(ctx context.Context, pod *corev1.Pod,
 				Name:            identityName,
 				Image:           fmt.Sprintf("%s:%s", zh.Config.Image, zh.Config.ImageVersion),
 				ImagePullPolicy: corev1.PullPolicy(zh.Config.ImagePullPolicy),
-				Args: []string{
-					"tproxy",
-					"-i",
-					fmt.Sprintf("%v.json", identityName),
-				},
+				Args: []string{"tproxy"},
```

Comment:
no need to specify identity name since it's always managed and invisible, so "ziti_id" default is fine for now.